README.md

```diff
@@ -21,14 +34,68 @@ $ helm repo add vmpooler-deployment https://puppetlabs.github.io/vmpooler-deploy
 
 ### Adding / updating charts
 
-```bash
-cd docs/
-helm package ../helm-charts/*
-helm repo index --url https://puppetlabs.github.io/vmpooler-deployment/ .
-git add .
-cd ..
-git commit -a
-```
+Make the desired changes to the helm chart in helm-charts/vmpooler and run `./update-chart.sh`.
+
+## Development
+
+Prerequisites:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+1. Become familiar with the configuration file `docker/vmpooler.yaml` as described in [VMPooler](https://github.com/puppetlabs/vmpooler) Core.
+   - The default configuration file only enables the dummy provider built into [VMPooler](https://github.com/puppetlabs/vmpooler) Core. See each project below for documentation on how to use provider specific options:
+      - [vmpooler-provider-ec2](https://github.com/puppetlabs/vmpooler-provider-ec2)
+      - [vmpooler-provider-gce](https://github.com/puppetlabs/vmpooler-provider-gce)
+      - [vmpooler-provider-vsphere](https://github.com/puppetlabs/vmpooler-provider-vsphere)
+2. Chose a Development method:
+   - Develop via local source:
+      1. Clone all of the known vmpooler projects under a common directory, for example:
```

Comment:
How do I find the list of `all the known vmpooler projects`? Can we have a script that does this?

```$ clone-known-vmpooler-projects```

Reply:
Oh darn, thought that I had listed those in the Readme, I'll add them.